src/OSPSuite.UI/Views/Charts/CurveMultiItemEditorView.cs

```diff
@@ -40,8 +40,7 @@ public override void InitializeBinding()
          base.InitializeBinding();
 
          _screenBinder.Bind(x => x.Color)
-            .To(colorPickEdit1)
-            .OnValueUpdated += colorChanged;
+            .To(colorPickEdit);
```

Comment:
Always rename the default control name to remove the 1 generated by default. I missed it previously